api/apiFunctions.ts

```diff
@@ -38,34 +40,53 @@ export async function registerAccount({
 }
 
 /**
- * Login to an existing account
- * @param props - The account data
- * @param props.email - The email of the user
- * @param props.password - The password of the user
- * @returns {Models.Session | Error} - The session object or an error
+ * Recover a User Password
+ * @param props - the props for the recover password function
+ * @param props.email - the email of the user
+ * @returns {Promise<IRecoveryToken>} - the recovery token
  */
-export async function loginAccount({
+export async function recoverPassword({
   email,
-  password,
-}: IAccountData): Promise<Models.Session | Error> {
+}: {
+  email: string;
+}): Promise<IRecoveryToken> {
+  const baseURL = getBaseURL();
+
   try {
-    return await account.createEmailPasswordSession(email, password);
+    const result = await account.createRecovery(
+      email,
+      `${baseURL}/account/recovery`,
+    );
+    return result;
   } catch (error) {
     console.error(error);
-    throw new Error('Error logging in user');
+    throw new Error('Error recovering password');
```

Comment:
If you just use `throw error` it will do everything we need. The stack track will tell us where it comes from. Then you can remove console.error(error) as well.

Reply:
✅ done.